packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -106,6 +106,18 @@ public void onProgressChanged(WebView view, int progress) {
     webView.getSettings().setDomStorageEnabled(true);
     webView.getSettings().setJavaScriptCanOpenWindowsAutomatically(true);
 
+    // Zoom support.
+    // // Zooms out the content to fit on screen by width.
+    webView.getSettings().setLoadWithOverviewMode(true);
+    // // It loads the WebView with the attributes defined in the meta tag of the webpage.
```

Comment:
The note format is not correctly.
```suggestion
    // It loads the WebView with the attributes defined in the meta tag of the webpage.
```